tests/loki/iofq_test.py

```diff
@@ -296,8 +277,7 @@ def test_beam_center_from_center_of_mass_is_close_to_verified_result():
 
 
 def test_phi_with_gravity():
-    params = make_params()
-    pipeline = sciline.Pipeline(loki_providers(), params=params)
+    pipeline = make_workflow()
     pipeline[BeamCenter] = _compute_beam_center()
     pipeline[CorrectForGravity] = False
     data_no_grav = pipeline.compute(
```

Comment:
I skimmed through the tests, it seemed it was mostly about refactoring by making a fixture that would create the workflows. I don't think there were fundamental changes to any of the tests.
Is that correct?

Reply:
Correct.